### DISCUSSION TITLE
TwirpError.php for common files (not for services)

### DISCUSSION THREAD

**Discussion by ilzrv** [twirphp/twirp#99] [Q&A] [answered]
I have created a common file named `location.proto`, which I would like to use in several services using `import` (`company.protobuf.Location location`). This file is not a service, just a description of the message.

After compilation, a file appears `TwirpError.php` (see screenshot), but it is not used anywhere. Should it be used somehow or is it a bug?

<img width="557" alt="Screen Shot 2022-01-20 at 16 49 26" src="https://user-images.githubusercontent.com/28765966/150352194-2e389b5d-3a84-4614-88b2-b17adf921e56.png">

**Comment by sagikazarmark** [comment, marked as answer]
TwirpError is used in services, but it generated individually since it needs to be generated once per package.

Since you package doesn't contain any services, there is an argument that probably TwirpError shouldn't be generated either. I think it should be fairly trivial thing to achieve.

**Reply by sagikazarmark** [reply]
Created an issue to track this.

**Reply by ilzrv** [reply]
Great! Thanks!
